library/std/src/panic.rs

```diff
@@ -11,7 +11,7 @@ use crate::thread::Result;
 
 #[doc(hidden)]
 #[unstable(feature = "edition_panic", issue = "none", reason = "use panic!() instead")]
-#[allow_internal_unstable(libstd_sys_internals, const_format_args, core_panic)]
+#[allow_internal_unstable(libstd_sys_internals, const_format_args, core_panic, rt)]
```

Comment:
necessary because panic items are invoked via `$crate::rt` and `rt` has it's own unstable feature that wasn't previously explicitly allowed